main/src/ui/contact_details/muc_config_form_provider.vala

```diff
@@ -63,12 +63,12 @@ public class MucConfigFormProvider : Plugins.ContactDetailsProvider, Object {
                     label = _("Occupants may change the subject");
                     break;
                 case "muc#roomconfig_whois":
-                    label = _("Discover real JIDs");
-                    desc = _("Who may discover real JIDs?");
+                    label = _("Permission to view JIDs");
+                    desc = _("Who is allowed to view the occupants' JIDs?");
                     break;
                 case "muc#roomconfig_roomsecret":
                     label = _("Password");
-                    desc = _("Password required for room entry, if any");
+                    desc = _("A password to restrict access to the room");
```

Comment:
I assumed that this is clear if this isn't a mandatory field, but if you want to add that information, I'd say:
"An optional password to restrict access to the room"

Reply:
That IMHO doesn't really get to the point. I'd slightly prefer "A password to restrict access to the room (empty = none)". I'm not too set on it tough and we can also take your initial suggestion.

Reply:
I don't understand how it misses the point. "Optional" means you can leave it empty and if no password is provided, there can be none set.

I'd like to avoid formulas with equal signs because they're technical sounding abbreviations which might not be understood by everyone.

If you're not convinced, here's a compromise:
"A password to restrict access to the room. Leave empty for free access"